doc/libpmemset/pmemset_source_from_file.3.md

```diff
@@ -29,8 +29,8 @@ _UW(pmemset_source_from_file)
 ```c
 #include <libpmemset.h>
 
-_UWFUNCR1(int, pmemset_source_from_file, struct pmemset_source **src, const char *file)
-_UWFUNCR1(int, pmemset_xsource_from_file, struct pmemset_source **src, const char *file, unsigned flags)
+_UWFUNCR2(int, pmemset_source_from_file, struct pmemset_source **src, *file)
```

Comment:
The result of this operation is:
```
int pmemset_source_from_fileU(struct pmemset_source **src, const char *file, );
int pmemset_source_from_fileW(struct pmemset_source **src, const wchar_t *file, );

```
You have to make sure, but I think you need to add a new macro for this particular case.